example/README.md

```diff
@@ -2,15 +2,98 @@
 
 Demonstrates how to use the blemulator plugin.
 
-## Getting Started
+## UI structure
 
-This project is a starting point for a Flutter application.
+App consists of two screens: `PeripheralListScreen` and 
+`PeripheralDetailsScreen`.
 
-A few resources to get you started if this is your first Flutter project:
+Skeleton UI flow:
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+![](./images/ui-flow.gif)
```

Comment:
A static map of screens would be more useful and lightweight.

Reply:
It has very low fps. And I think it should be smaller. I don't know if you know, but you can use `<img />` tag with `width` attribute to set size, so the GIF image does not need to be downscaled itself.